lib/views/internallogin_screen.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/main.dart';
+
+class InternalloginScreen extends ConsumerWidget {
+  const InternalloginScreen({super.key});
+
+  Future<void> handleBasicLogin(
```

Comment:
Seems to be a case for ViewModel

Reply:
This will be done in this [branche](https://github.com/TUM-Dev/gocast_mobile/tree/34-clean-code)